owncloudApp/src/androidTest/java/com/owncloud/android/shares/ui/PublicShareEditionDialogFragmentTest.kt

```diff
@@ -32,33 +30,31 @@ import androidx.test.espresso.matcher.ViewMatchers.withText
 import androidx.test.ext.junit.runners.AndroidJUnit4
 import androidx.test.rule.ActivityTestRule
 import com.owncloud.android.R
-import com.owncloud.android.capabilities.db.OCCapability
-import com.owncloud.android.capabilities.viewmodel.OCCapabilityViewModel
 import com.owncloud.android.datamodel.OCFile
 import com.owncloud.android.lib.resources.shares.RemoteShare
 import com.owncloud.android.shares.db.OCShare
 import com.owncloud.android.shares.ui.fragment.PublicShareDialogFragment
-import com.owncloud.android.shares.viewmodel.OCShareViewModel
-import com.owncloud.android.utils.ViewModelUtil
-import com.owncloud.android.vo.Resource
+import com.owncloud.android.utils.TestUtil
 import org.junit.Before
 import org.junit.Rule
 import org.junit.Test
 import org.junit.runner.RunWith
 import org.mockito.Mockito.`when`
 import org.mockito.Mockito.mock
+import java.text.DateFormat
+import java.text.SimpleDateFormat
+import java.util.GregorianCalendar
+import java.util.TimeZone
 
 @RunWith(AndroidJUnit4::class)
 class PublicShareEditionDialogFragmentTest {
     @Rule
     @JvmField
     val activityRule = ActivityTestRule(TestShareFileActivity::class.java, true, true)
 
-    private val account = mock(Account::class.java)
-    private val capabilitiesLiveData = MutableLiveData<Resource<OCCapability>>()
-    private val sharesLiveData = MutableLiveData<Resource<List<OCShare>>>()
     private val file = mock(OCFile::class.java)
     private val publicShare = mock(OCShare::class.java)
+    private val expirationDate = 1556575200000
```

Comment:
When is it?

Reply:
I put a comment with the date, but do you think is really important?

Reply:
I don't know if it will some years later or some years ago.... I don't know... 
The problem is if the date is a coming date, it will fail tests just because we are in that date or it's past... 
I would do it that date dynamically in order to avoid that issue. For example:
`System. currentTimeMillis() + 2000000`

Reply:
In this test we are setting a date in a view and checking if it appears, so I think is not affected by a coming or past date

Reply:
ok